install/kubernetes/Makefile

```diff
@@ -35,7 +35,7 @@ LOGO_PATH := Documentation/images/logo-solo.svg
 define generate_values_yaml
 echo -e "# File generated by $(RELATIVE_DIR)/Makefile; DO NOT EDIT." > $(2); \
 echo -e "# This file is based on $(RELATIVE_DIR)/cilium/$(VALUES_TMPL_FILE_NAME).\n" >> $(2); \
-cat $(1) | envsubst >> $(2)
+yq ea '. as $$item ireduce ({}; . * $$item )' $(1) | envsubst >> $(2)
```

Comment:
This requires yq to be installed on the developer's machine. There are two popular utilities called `yq` (https://github.com/kislyuk/yq and https://github.com/mikefarah/yq), which one do we require?

Given that both contributors and backporters often need to regenerate the `values.yaml`, can we make this easier on developers somehow? 

I think we should at least add it to dev-doctor. But preferably maybe even run a yq` docker image here, since `yq` is not a tool installed on many default installations. Or have a custom go tool in `tools/` which does YAML unification for us.

Reply:
I'm using https://github.com/mikefarah/yq/, but good idea on adding it to `dev-doctor`/using a docker image. I'll try switching to a docker image.

I don't think the `tools/` approach will work well unless we vendor `yq` or something though, it can't be naive or we'll lose the YAML comments and the structure/ordering will not be preserved as well.